core/genesis_test.go

```diff
@@ -95,14 +95,14 @@ func TestSetupGenesis(t *testing.T) {
 			wantConfig: customg.Config,
 		},
 		{
-			name: "custom block in DB, genesis == testnet",
+			name: "custom block in DB, genesis == baklava",
```

Comment:
For the sake of completion, should there be a test case for the alfajores genesis here?